src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -158,6 +159,10 @@ export class PatientViewPageStore {
         return this.pageMode === 'sample' ? this.sampleId : this.patientId;
     }
 
+    @computed get userName(){
+		return getBrowserWindow().globalStores.appStore.userName;

```

Comment:
This this the full email address?

Reply:
@jjgao  Yes, it is the whole email address. The format is like username@mskcc.org.

Reply:
Maybe I should change the name to the userEmail or something?

Reply:
that's ok. Maybe add a comment.